src/axom/primal/operators/detail/in_curved_polygon_impl.hpp

```diff
@@ -183,28 +183,27 @@ double adaptive_winding_number(const Point<T, 2>& q,
   if(ord <= 0) return 0.0;  // Catch degenerate cases
 
   // Use linearity as base case for recursion
-  if(c.isLinear(EPS))
-    return 0.0 - linear_winding_number(q, c[0], c[ord], edge_tol);
+  if(c.isLinear(EPS)) return linear_winding_number(q, c[0], c[ord], edge_tol);
 
   Polygon<T, 2> controlPolygon(c.getControlPoints());
 
-  // If q is outside the control polygon, for an open Bezier curve, the winding
-  //  number for the shape connected at the endpoints with straight lines is zero.
-  //  We then subtract the contribution of this line segment.
-  if(!in_polygon(q, controlPolygon, true, false, EPS))
-    return 0.0 - linear_winding_number(q, c[0], c[ord], edge_tol);
-
   // Check if our new curve is convex.
   //  If so, all subcurves will be convex as well
   if(!isConvexControlPolygon)
     isConvexControlPolygon = is_convex(controlPolygon, EPS);
```

Comment:
Minor:  (To more clearly distinguish between the `if` and `else` clauses) 

```suggestion
  if(!isConvexControlPolygon)
  {
    isConvexControlPolygon = is_convex(controlPolygon, EPS);
  }
```